contribute.mdx

```diff
@@ -35,106 +35,77 @@ Get paid for the content you write that we publish. You can get up to €200 dis
 ## How to get started
```

Comment:
Hero image not showed (white square), here on local build:
![Capture d’écran 2022-10-19 à 9 44 42 AM](https://user-images.githubusercontent.com/341803/196628659-db522455-4b76-47a7-a08f-9bf63bc8cf64.png)

It should be used with srcset stuffs, here in prod:

![Capture d’écran 2022-10-19 à 9 45 12 AM](https://user-images.githubusercontent.com/341803/196628865-480f0e13-2a89-4625-bd55-9120be5dd16f.png)